tests/integration/simulacron/test_cluster.py

```diff
@@ -16,14 +16,20 @@
 except ImportError:
     import unittest  # noqa
 
-from tests.integration.simulacron import SimulacronCluster
-from tests.integration import (requiressimulacron, PROTOCOL_VERSION)
-from tests.integration.simulacron.utils import prime_query
+import logging
+from packaging.version import Version
+
+import cassandra
+from tests.integration.simulacron import SimulacronCluster, SimulacronBase
+from tests.integration import (requiressimulacron, PROTOCOL_VERSION, DSE_VERSION, MockLoggingHandler)
+from tests.integration.simulacron.utils import prime_query, start_and_prime_singledc
 
 from cassandra import (WriteTimeout, WriteType,
                        ConsistencyLevel, UnresolvableContactPoints)
-from cassandra.cluster import Cluster
+from cassandra.cluster import Cluster, ControlConnection
+
 
+PROTOCOL_VERSION = min(4, PROTOCOL_VERSION if (DSE_VERSION is None or DSE_VERSION >= Version('5.0')) else 3)
```

Comment:
this is only a default value?

Reply:
Stole it from another simulacron test after getting protocol errors. Looks like simulacron barfs when we try to pass a higher protocol version.